> > The tests were failing on the unique field inside of peewee migrator.py regardless of what I put
> 
> Can I have more details?

With what was suggested above this error occurs:

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/peewee_migrate/router.py", line 194, in run_one
    migrate(migrator, self.database, fake=fake)
  File "<string>", line 31, in migrate
  File "/usr/local/lib/python3.9/dist-packages/peewee_migrate/migrator.py", line 128, in wrapper
    return method(migrator, model, *args, **kwargs)
  File "/usr/local/lib/python3.9/dist-packages/peewee_migrate/migrator.py", line 348, in add_index
    field.unique = unique
AttributeError: 'NoneType' object has no attribute 'unique'
```

When I follow the [peewee docs](https://docs.peewee-orm.com/en/latest/peewee/playhouse.html?highlight=migrator#supported-operations), this occurs:

```
[2023-06-08 07:05:37] peewee_migrate                 ERROR   : Migration failed: 017_update_indexes
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/peewee_migrate/router.py", line 194, in run_one
    migrate(migrator, self.database, fake=fake)
  File "<string>", line 31, in migrate
  File "/usr/local/lib/python3.9/dist-packages/peewee_migrate/migrator.py", line 127, in wrapper
    return method(migrator, migrator.orm[model], *args, **kwargs)
KeyError: 'recordings'
```

--- a/frigate/app.py
+++ b/frigate/app.py
@@ -172,7 +172,7 @@ class FrigateApp:
             db.execute_sql("VACUUM;")
 
             try:
-                with open("/config/.vacuum", "w") as f:
+                with open(f"{CONFIG_DIR}/.vacuum", "w") as f:
                     f.write(str(datetime.datetime.now().timestamp()))
             except PermissionError:
                 logger.error("Unable to write to /config to save DB state")
@@ -193,8 +193,8 @@ class FrigateApp:
         router.run()
 
         # check if vacuum needs to be run
-        if os.path.exists("/config/.vacuum"):
-            with open("/config/.vacuum") as f:
+        if os.path.exists(f"{CONFIG_DIR}/.vacuum"):
+            with open(f"{CONFIG_DIR}/.vacuum") as f:
                 try:
                     timestamp = int(f.readline())
                 except Exception:

